config/application.rb

```diff
@@ -38,9 +44,6 @@ class Application < Rails::Application
     # Use the responders controller from the responders gem
     config.app_generators.scaffold_controller :responders_controller
 
-    # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
-
```

Comment:
while this change makes sense, did you check that this will not change existing configuration? How about adding the 7.1 defaults and overriding differences with 5.1 with individual statements. I see that current PR is wip but decided to comment.

Reply:
Yes, definitely makes sense changing this to `config.load_defaults 7.0`

Right now I'm in the process of comparing the configs to see if we need to override any or get rid of customizations in the code due to know defaults. Yes, this is WIP :P

I just pushed mainly for myself, because I like GitHub's diff. But early feedback is useful too.